src/Xamarin.Android.NUnitLite/Gui/AndroidRunner.cs

```diff
@@ -381,7 +381,7 @@ internal void AddTestFilters (IEnumerable<string> included, IEnumerable<string>
 		static void ChainCategoryFilter (IEnumerable <string> categories, bool negate, ref TestFilter chain)
 		{
 			bool gotCategories = false;
-			if (categories != null) {
+			if (categories != null && categories.Any ()) {
```

Comment:
We need to check `Any()`, because an empty `IEnumerable` should also be skipped

Reply:
It should arguably be skipped, *but* it means that `categories` will be enumerated multiple times, which is potentially problematic.

It's probably better to not use the `.Any()`, and have an alternate check.